system/languages/en.php

```diff
@@ -307,6 +307,10 @@
 		"art-add-comment" => "add comment",
 		"art-comment" => "comment",
 		"art-comments" => "comments",
+		"art-title" => "title",
+		"art-continue-discussion" => "contribute",
```

Comment:
`contribute` seems weird here, it does not match the string label, nor the Dutch translation. Shouldn't this just be `Continue discussion`?

Reply:
No, I ment "contribute to **this** discussion"

Reply:
In that case, perhaps the id of the string should be changed to "art-contribute" or something similar?